content/en/security/cloud_security_management/agentless_scanning/_index.md

```diff
@@ -43,6 +43,8 @@ The following table provides a summary of Agentless scanning technologies in rel
 | Serverless                  | AWS, AWS Lambda                                             |
 | Serverless languages        | .Net, Python, Java, Ruby, Node.js, Go                        |
 
+**Note**: AMIs must be stored in an account that uses Datadog's AWS integration. Otherwise, Datadog can't read its underlying Amazon Elastic Block Store (EBS) snapshot, so it can't scan or report on the AMI.
```

Comment:
```suggestion
**Note**: AMIs must be stored in an account that uses Datadog's AWS integration. Otherwise, Datadog can't read the AMI's underlying Amazon Elastic Block Store (EBS) snapshot, so it can't scan or report on the AMI.
```
I'm not sure I've corrected this properly in terms of meaning, just trying to avoid two "it" references that refer to different nouns.